pcawg_metadata_parser/pc_report-sanger_summary_counts.py

```diff
@@ -98,13 +100,20 @@ def compute_site_report_new(metadata_dir, report_dir, today_donors):
         'Total': len(today_donors[0].union(today_donors[1]).difference(site_assigned_donors))
     }
 
+    unassigned_uncalled_donors = today_donors[0].difference(site_assigned_donors)
+
     # today's counts
     with open(report_dir + '/summary_compute_site_counts.json', 'w') as o: o.write(json.dumps(site_summary))
 
-    # get all previous days counts
-    [dates, metadata_dirs] = get_metadata_dirs(metadata_dir, '2015-03-07')
+    with open(report_dir + '/unassigned_uncalled_donors.txt', 'w') as o:
+        o.write('# Unassigned and uncalled donors\n')
+        o.write('# dcc_project_code' + '\t' + 'submitter_donor_id' + '\n')
+        o.write('\n'.join(unassigned_uncalled_donors) + '\n') 
 
     """
+    # get all previous days counts
+    [dates, metadata_dirs] = get_metadata_dirs(metadata_dir, '2015-04-01')
```

Comment:
Any reason the date for start tracking Sanger variant calling needs to be updated?